plugins/package-managers/maven/src/test/kotlin/utils/MavenDependencyHandlerTest.kt

```diff
@@ -219,7 +222,24 @@ class MavenDependencyHandlerTest : WordSpec({
             every { dependency.repositories } returns repos
             every { handler.support.parsePackage(artifact, repos) } throws exception
 
-            handler.createPackage(dependency, issues) should beNull()
+            val pkg = handler.createPackage(dependency, issues)
+            pkg.shouldNotBeNull()
+
+            with(pkg.id) {
+                type shouldBe "Maven"
+                namespace shouldBe artifact.groupId
+                name shouldBe artifact.artifactId
+                version shouldBe artifact.version
+            }
+
+            with(pkg) {
+                binaryArtifact shouldBe RemoteArtifact.EMPTY
+                declaredLicenses should beEmpty()
+                description shouldBe ""
+                homepageUrl shouldBe ""
```

Comment:
This should be `should beEmpty()` similar to like one line above, but it requires to import either version via an alias to avoid a name clash. Probably import the line above like `import io.kotest.matchers.collections.beEmpty as beEmptyCollection`.